test/integration/azureSpecialProperties.spec.ts

```diff
@@ -232,26 +241,30 @@ describe("AzureSpecialProperties", () => {
           customHeaders: {
             "x-ms-client-request-id": validClientId
           }
-        }
+        },
+        ...responseStatusChecker
       };
-      let result = await client.xMsClientRequestId.get(options);
-      assert.equal(result._response.status, 200);
-      assert.equal(result._response.headers.get("x-ms-request-id"), "123");
+      await client.xMsClientRequestId.get(options);
```

Comment:
Here we are missing the test for "x-ms-request-id" equality to "123"

Reply:
Fixed.